lib/osf-components/addon/components/make-decision-dropdown/component.ts

```diff
@@ -271,5 +310,29 @@ export default class MakeDecisionDropdown extends Component<Args> {
     reset() {
         this.decisionTrigger = undefined;
         this.comment = undefined;
+        this.changeset.rollback();
```

Comment:
One caution on a rollback, if you try it after an unsuccessful save (such as from the API returning an error), in my experience the changeset will already be executed onto the object, and a rollback will have no effect. In that situation (around L292), you might have to reload the info from the database.

Reply:
Ahh, good call. I will see how it behaves after an error and update accordingly

Reply:
It may be fine because you're setting those two fields to undefined in L311-312.

Reply:
Yea, it looks like it rolls back just fine after an error!